testsuite/openshift/objects/auth_config.py

```diff
@@ -151,8 +151,30 @@ def add_opa_policy(self, name, rego_policy):
             }
         })
 
+    @modify
+    def add_external_registry(self, name, endpoint, ttl=0):
+        """
+        Adds OPA policy that is declared as an HTTP endpoint
+        """
+        policy = self.model.spec.setdefault("authorization", [])
+        policy.append({
+            "name": name,
+            "opa": {
+                "externalRegistry": {
+                    "endpoint": endpoint,
+                    "ttl": ttl
+                }
+            }
+        })
+
+    @modify
+    def remove_all_authorization(self):
```

Comment:
Where is this used?

Reply:
This was intended to use with test, where the external registry URL will be swapped to a different endpoint. But I scraped this test case later on.